File: workspaces/cWorkspace/fizzbuzz.c
#include <stdio.h>

void main(){
	int i = 1;
	for ( i ; i <= 100 ; i++ ){
		if( i % 3 == 0 )
			{printf("fizz\n");}
		if( i % 5 == 0 )
			{printf("buzz\n");}
		if(!( i % 5 == 0 ) && !( i % 3 == 0 ))
			{printf("%d\n",i);}
	}
}

File: workspaces/pySpace/fizzBuzz.py
"""

	Simple program for fizzing your buzz

"""
# E N T E R P R I S E L E V E L V A R I A B L E D E C L A R A T I O N
fizz='fizz'
buzz='buzz'

for num in range(1,100):
	if num % 3 is 0:
		print(fizz)
	if num % 5 is 0:
		print(buzz)
	if not (num % 3 is 0) and not (num % 5 is 0):
		print(num)
